custom/opm/health_status.py

```diff
@@ -160,15 +160,14 @@ class AWCHealthStatus(object):
           "attended at least one growth monitoring session in the last 3 "
           "months.  Exempt if no scale was available at the VHND."),
          'child_mult_3_months'),
-        # TODO ors is hard, skipping for now
```

Comment:
@esoergel assuming you have commented this, did I miss anything in this implementation?